lms/djangoapps/mobile_api/settings.py

```diff
@@ -0,0 +1,10 @@
+'''
+	Settings for Mobile Api
+'''
+
+FB_SETTINGS = {	'_FACEBOOK_APP_SECRET' : "6c26348ef355f53a531890e980ddc731",
+				'_FACEBOOK_APP_ID' : "735343629893573",
+				'_FACEBOOK_API_VERSION' : "v2.2/"}
```

Comment:
Nitpick: For readability of the client code that use this version constant, it should not include the ending `/`.  Unless you want to rename it to `_FACEBOOK_API_VERSION_WITH_SLASH`.  :)